### PR TITLE
Finish #1998

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | all |
| Fixed tickets | replaces #1998 |

### CODE REVIEW

**Comment by WouterJ** [comment]
@xabbuh can you please update this PR with the change proposed by @stof? (see also the blogpost referenced by @webda2l)

Btw, can you please also handle #4797. It explains the same thing, but less verbose and in another location. Would be great if we can make both PRs ready to merge!

**Comment by WouterJ** [comment]
@xabbuh can you please take care of this one and #4797?

File: cookbook/form/form_collections.rst [hunk 1]
```diff
@@ -729,5 +729,75 @@ the relationship between the removed ``Tag`` and ``Task`` object.
...
+            <ul {{ block('widget_container_attributes') }}>
+                {% for child in form %}
+                    {{ form_row(child) }}
+                {% endfor %}
+            </ul>
+        {% endblock %}
+
+    .. code-block:: html+php
+
+        <!-- src/AppBundle/Resources/views/Form/_tasks_widget.html.php -->
```
Comment (xabbuh) [inline cookbook/form/form_collections.rst]: Actually, I'm stuck how this should look like in the PHP template.

**Comment by xabbuh** [comment]
@WouterJ Do we really need #4797 at all when this is merged? Imho everything should be fine then. The only thing I can image is to add a cross-reference to the other article. What do you think?

**Comment by WouterJ** [comment]
@xabbuh it's up to you :) I think the location of #4797 is better and the form collections article should just reference.

**Comment by WouterJ** [comment]
Closing in favor of https://github.com/symfony/symfony-docs/pull/5643

**Pull request closed without merging**